packages/medusa/src/api/routes/admin/shipping-options/index.ts

```diff
@@ -55,6 +55,9 @@ export const defaultRelations = ["region", "profile", "requirements"]
 /**
  * @schema AdminShippingOptionsListRes
  * type: object
+ * required:
+ *   - shipping_options
+ *   - count
```

Comment:
I think we should also have here `offset` and `limit` but they're also missing from the response properties.

Reply:
Good catch 🎣 . Fixed in 5c3615b473681bd2265b434cb935f335c2840436
I also added the missing `required` on the other schemas.